beater/http.go

```diff
@@ -130,6 +160,7 @@ func (h *httpServer) start() error {
 	}
 
 	if h.cfg.MaxConnections > 0 {
+		// TODO: reload
```

Comment:
There is some configuration in `httpServer.start()` that is not executed elsewhere, such as how to listen (`Listen` vs `ListenTLS`), which probably would necessitate a server restart.